rust_dev_preview/eks/README.md

```diff
@@ -0,0 +1,70 @@
+# AWS SDK for Rust code examples for Amazon EKS
+
+## Purpose
+
+These examples demonstrate how to perform several Amazon Elastic Kubernetes Service (Amazon EKS) operations using the developer preview version of the AWS SDK for Rust.
+
+Amazon EKS is a managed service that makes it easy for you to run Kubernetes on AWS without needing to install and operate your own Kubernetes clusters.
+
+## Code examples
+
+- [Create and delete a cluster](src/bin/create-delete-cluster.rs) (CreateCluster DeleteCluster)
+- [List clusters](src/bin/list-clusters.rs) (ListClusters)
+
+## ⚠ Important
+
+- We recommend that you grant this code least privilege, 
+  or at most the minimum permissions required to perform the task.
+  For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the AWS Identity and Access Management User Guide.
+- This code has not been tested in all AWS Regions.
+  Some AWS services are available only in specific
+  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+- Running this code might result in charges to your AWS account.
+
+## Running the code examples
+
+### Prerequisites
+
+You must have an AWS account, and have configured your default credentials and AWS Region as described in [https://github.com/awslabs/aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust).
```

Comment:
Mention the ARN requirement and how to set up that ARN here.

Reply:
Done.